compass/ocean/tests/utility/combine_topo/__init__.py

```diff
@@ -55,101 +77,101 @@ def setup(self):
 
         config = self.config
         section = config['combine_topo']
+
+        # Get input filenames and resolution
         antarctic_filename = section.get('antarctic_filename')
-        self.add_input_file(filename=antarctic_filename,
-                            target=antarctic_filename,
-                            database='bathymetry_database')
         global_filename = section.get('global_filename')
-        self.add_input_file(filename=global_filename,
-                            target=global_filename,
-                            database='bathymetry_database')
-
-        cobined_filename = section.get('cobined_filename')
-        self.add_output_file(filename=cobined_filename)
 
+        # Parse resolution and assign resolution attributes
+        if self.target_grid == 'cubed_sphere':
+            self.resolution = section.getint('resolution_cubedsphere')
+            self.resolution_name = f'ne{self.resolution}'
+        elif self.target_grid == 'lat_lon':
+            self.resolution = section.getfloat('resolution_latlon')
+            self.resolution_name = f'{self.resolution:.4f}_degree'
+
+        # Build output filenames
+        datestamp = datetime.now().strftime('%Y%m%d')
+        scrip_filename = f'{self.resolution_name}_{datestamp}.scrip.nc'
+        combined_filename = '_'.join([
+            antarctic_filename.strip('.nc'),
+            global_filename.strip('.nc'),
```

Comment:
For consistency with existing filenames:
```suggestion
            antarctic_filename.strip('.nc'),
            'and',
            global_filename.strip('.nc'),
```

Reply:
Hmm, I see that this still isn't consistent with other filename because we also converted dashes to underscores, so probably not worth the trouble.